docs/installation.md

```diff
@@ -22,6 +22,26 @@ $ nimble install owlkettle@#head
   pkg-config --libs gtk4
   ```
 
+### Properly setting up MSYS2 on Windows
+
+If you want to make sure you have the `pkg-config` in your system, 
```

Comment:
`pkg-config` is a package right?

In that case it would be "If you want to make sure that you have the `pkg-config` package in your system, you need to install your MSYS2's environment toolchain.

Reply:
No, it's not. `pkg-config` is the binary itself, the package are: `mingw-w64-x86_64-toolchain`, `mingw-w64-ucrt-x86_64-pkg-config` and `mingw-w64-clang-x86_64-pkg-config`.

Reply:
In that case my bad, I'd mark this comment chains as resolved if I could :sweat_smile:

Reply:
Ok! No problem.